cvat/apps/events/views.py

```diff
@@ -44,6 +48,26 @@ def create(self, request):
 
         return Response(serializer.validated_data, status=status.HTTP_201_CREATED)
 
+    @extend_schema(summary='Log external events',
```

Comment:
Do we really need several end-points to send events (add a row into Clickhouse)?

Reply:
reused existing endpoint